libraries/AP_RSSI/AP_RSSI.cpp

```diff
@@ -187,8 +187,12 @@ float AP_RSSI::read_pin_rssi()
 // read the RSSI value from a PWM value on a RC channel
 float AP_RSSI::read_channel_rssi()
 {
+
+    // true if an rc input was received recently (does not consider rc overrides)
+    bool recent_rc_receiver_input = rc().get_fs_timeout_ms() > (AP_HAL::millis() - rc().last_rc_receiver_input_ms());
```

Comment:
```suggestion
    bool recent_rc_receiver_input = rc().get_fs_timeout_ms() > (AP_HAL::millis() - rc().last_input_ms());
```
RC_Channels already has the method you need. You should be able to drop the RC_Channels changes.

Reply:
My concern with rc().last_input_ms() is that it also counts RC overrides as an input.  So even if the receiver link is lost, that value will still update if overrides are being sent (and consequently RSSI would not go to zero).

I generally try to avoid using RC overrides, but there are some cases where its hard to avoid.  I noticed a while back that you can't detect RC failsafe / lost link if using overrides (I opened issue #21610 a while back to try to address this).

By not counting RC overrides as input for setting RSSI to zero, a companion computer could at least keep an eye on RSSI to understand if the RC link is lost when using overrides.

Reply:
OK, in that case you should add comments explaining the difference between the two methods.